portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,12 +30,19 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data. I have to redo everything from step 5*/
+/** Servlet that gets and stores user comment data.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        //Check if the quantity is null. This usually happens when the user leave the quantity input blank
+
+        if (request.getParameter("quantity") == null) {
+            Integer quantity = 5;
```

Comment:
This has no effect. If quantity is null, it creates a temporary variable and then deletes it.

Reply:
Good to know, I'll just remove that conditional.

Reply:
What does `Integer.parseInt(null)` do? set it to zero? Does that work for you?